networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,38 @@ The Network Observability Operator is now stable and the release channel is upgr
+
+* Previously, network flows between pods running on different nodes were sometimes not correctly identified as being duplicates because they are captured by different network interfaces. This resulted in over-estimated metrics displayed in the console plugin. Now, flows are correctly identified as duplicates, and the console plugin displays accurate metrics. (link:https://issues.redhat.com/browse/NETOBSERV-755[NETOBSERV-755])
+
+* Previously, the "reporter" option in the console plugin, which is used to filter flows based on the observation point of either source node or destination node, mixed the flows regardless of the node observation point. This was due to network flows being incorrectly reported as Ingress or Egress at the node level.  Now, the network flow direction reporting is corrected. The "reporter" option filters for source observation point, or destination observation point, as expected. (link:https://issues.redhat.com/browse/NETOBSERV-696[NETOBSERV-696])
+
+* Previously, for agents configured to send flows directly to the processor as GRPC+protobuf requests, the submitted payload could be too large and is rejected by the processors' GRPC server. This occured under  very-high-load scenarios and with only some configurations of the agent. The agent logged an error message, such as: _grpc: received message larger than max_. As a consequence, there was information loss about those flows.
+Now, the GRPC payload is split into several messages when the size exceeds a threshold. As a result, the connectivity is maintained. (link:https://issues.redhat.com/browse/NETOBSERV-617[NETOBSERV-617])
+
+* Previously, you could install the Network Observability Operator using a custom namespace. This release introduces the `conversion webhook` which changes the CSV. Because of this change, all the available namespaces are no longer listed. Additionally, to enable Operator metrics collection, namespaces that are shared with other Operators, like the `openshift-operators` namespace, cannot be used. Now, the Operator must be installed in the `openshift-netobserv-operator` namespace. You cannot automatically upgrade to the new Operator version if you previously installed the Network Observability Operator using a custom namespace. If you previously installed the Operator using a custom namespace, you must delete the instance of the Operator that was installed and re-install your operator in the `openshift-netobserv-operator` namespace. It is important to note that custom namespaces, such as the commonly used `netobserv` namespace, are still possible for the `FlowCollector`, Loki, Kafka, and other plugins. (link:https://issues.redhat.com/browse/NETOBSERV-907[NETOBSERV-907])(link:https://https://issues.redhat.com/browse/NETOBSERV-956[NETOBSERV-956])
```

Comment:
rewording suggestion and Operator capitalization:
```suggestion
* Previously, you could install the Network Observability Operator using a custom namespace. This release introduces the `conversion webhook` which changes the CSV. Because of this change, all the available namespaces are no longer listed. Additionally, namespaces that are shared with other Operators to enable Operator metrics collection, like the `openshift-operators` namespace, cannot be used. Now, the Operator must be installed in the `openshift-netobserv-operator` namespace. You cannot automatically upgrade to the new Operator version if you previously installed the Network Observability Operator using a custom namespace. If you previously installed the Operator using a custom namespace, you must delete the instance of the Operator that was installed and re-install your Operator in the `openshift-netobserv-operator` namespace. It is important to note that custom namespaces, such as the commonly used `netobserv` namespace, are still possible for the `FlowCollector`, Loki, Kafka, and other plug-ins. (link:https://issues.redhat.com/browse/NETOBSERV-907[NETOBSERV-907])(link:https://https://issues.redhat.com/browse/NETOBSERV-956[NETOBSERV-956])
```
additionally, what does CSV mean in this context? ISG recommends expanding the abbreviation unless it's commonly known

Reply:
`ClusterServiceVersion` yeah probs not known..I had to verify this myself. Good catch.